test/integration/targets/copy/tasks/main.yml

```diff
@@ -588,36 +590,38 @@
 #
 # Recursive copy of tricky symlinks
 #
-- name: Create a directory to copy from
-  file:
-    path: '{{ output_dir }}/source1'
-    state: directory
-
-- name: Create a directory outside of the tree
-  file:
-    path: '{{ output_dir }}/source2'
-    state: directory
-
-- name: Create a symlink to a directory outside of the tree
-  file:
-    path: '{{ output_dir }}/source1/link'
-    src: '{{ output_dir }}/source2'
-    state: link
-
-- name: Create a circular link back to the tree
-  file:
-    path: '{{ output_dir }}/source2/circle'
-    src: '../source1'
-    state: link
-
-- name: Create output directory
-  file:
-    path: '{{ output_dir }}/dest1'
-    state: directory
+- block:
+  - name: Create a directory to copy from
+    file:
+      path: '{{ role_path }}/tmp/source1'
```

Comment:
I don't think role_path is an appropriate location to be creating more files.  Doesn't a later commit switch us to using a different user?  Maybe tmpdir (/tmp/ or /var/tmp/) is the only choice that makes sense here?